src/storage/hummock_sdk/src/key_range.rs

```diff
@@ -22,26 +22,17 @@ use super::version_cmp::VersionedComparator;
 pub struct KeyRange {
     pub left: Bytes,
     pub right: Bytes,
-    // FIXME: Note that currently `inf` is not always correctly set throughout the codebase,
-    // e.g. there is usage like KeyRange::new(Bytes::new(), Bytes::new()).
-    // It doesn't affect correctness only because we don't use `inf` anywhere for now.
-    pub inf: bool,
 }
 
 impl KeyRange {
```

Comment:
shall we use `start_bound_inf` and `end_bound_inf` function to shadow the check logic like `self.right.is_empty()`  which make it more readable

```rust
fn start_bound_inf(&self) -> bool {
  self.left.is_empty()
}
```

Reply:
good idea